mlflow/deployments/cli.py

```diff
@@ -220,3 +237,24 @@ def run_local(flavor, model_uri, target, name, config):
     """
     config_dict = _user_args_to_dict(config)
     interface.run_local(target, name, model_uri, flavor, config_dict)
+
+
+@commands.command("predict")
+@parse_custom_arguments
+@deployment_name
+@target_details
+@parse_input
+@parse_output
+def predict(target, name, input_path, output_path, config):
```

Comment:
```suggestion
def predict(target, name, input_path, output_path):
```

Can we remove the `config` argument as described in https://github.com/mlflow/mlflow/pull/3597/files#r513748823? Assuming it's only needed to specify a specific model version to score.